package/MDAnalysis/core/AtomGroup.py

```diff
@@ -457,7 +457,84 @@
 # And the return route
 _SINGULAR_PROPERTIES = {v: k for k, v in _PLURAL_PROPERTIES.items()}
 
-_FIFTEEN_DEPRECATION = "This will be removed in version 0.15.0"
+_SIXTEEN_DEPRECATION = "This will be removed in version 0.16.0"
```

Comment:
On 31 Mar, 2016, at 11:04, David Dotson wrote:

> The reason this went from "FIFTEEN" to "SIXTEEN" was because we were originally planning for the topology refactor to land in 0.15.0, with the release we're working toward now being a point release (0.14.1).

Er, I think the next release WILL be 0.15.0... we already have too much new stuff (see CHANGELOG). It's just the current version tag is still 0.14.1.

--
Oliver Beckstein * orbeckst@gmx.net
skype: orbeckst  * orbeckst@gmail.com